### PR TITLE
feat(icons) add new moderator icon

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: react/features/filmstrip/components/native/ModeratorIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+// eslint-disable-next-line lines-around-comment
+// @ts-ignore
+import { BaseIndicator } from '../../../base/react';
+
+/**
+ * Thumbnail badge showing that the participant is a conference moderator.
+ *
+ * @returns {JSX.Element}
+ */
+export default function MoeratorIndicator(): JSX.Element {
```
Comment (horymury) [inline react/features/filmstrip/components/native/ModeratorIndicator.tsx:13]: typo in `MoeratorIndicator`

Comment (horymury) [reply react/features/filmstrip/components/native/ModeratorIndicator.tsx:13]: also do we need named function here?

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/filmstrip/components/native/ModeratorIndicator.tsx (hunk 1 above)
Comment (saghul) [reply react/features/filmstrip/components/native/ModeratorIndicator.tsx:13]: Not necessarily, it just serves as "documentation".

**Review by horymury — APPROVED** [review]

**Review by horymury — APPROVED** [review]

**Comment by robertpin** [comment]
Why did we change this? It's been a crown for a very long time and the new one doesn't really align with anything else in the design. @saghul

**Comment by saghul** [comment]
We received requests to change it away from the crown. I requested a design and after some back and forth this is what we got.

**Comment by robertpin** [comment]
Not that it's worth anything at this point, but I'm against it. I think it looks kinda ugly and I don't see the benefit. For someone that's never used Jitsi when they first see a crown or an "M" in a box, it doesn't mean anything. And after you hover it and see "Moderator" in the tooltip you learn it and never have to do it again.

**Pull request merged**